ac/ac-stroop/src/ActivityRunner.jsx

```diff
@@ -89,22 +89,38 @@ const Start = withState('ready', 'setReady', false)(
 
 let noAnswerTimeout;
 let delayTimeout;
+let timeCounter;
 
-const Delay = ({ next, delay }) => {
+const CountDownTimer = TimedComponent(() => {
+  timeCounter -= 1000;
+
+  return <div style={styles.text}>{timeCounter / 1000 + 1 + ' s'}</div>;
+}, 1000);
+
+const Delay = ({ next, delay, props }) => {
   clearTimeout(delayTimeout);
   delayTimeout = setTimeout(next, delay);
-  return <div style={styles.text}>Waiting for next question</div>;
+  timeCounter = delay;
+  return (
+    <React.Fragment>
+      <div style={styles.text}>Waiting for next question</div>
+      <CountDownTimer {...props} />
```

Comment:
Here you give properties to the component `CountDownTimer` but this component does not use any props

Reply:
Fixed.